src/families/ethereum/types.js

```diff
@@ -234,6 +235,9 @@ export const reflect = (declare: (string, Spec) => void) => {
       },
       getEstimatedGasLimit: {
         returns: "BigInt"
+      },
+      getERC20Balances: {
+        returns: ["CoreBigInt"]
```

Comment:
`["BigInt"]` ?

Reply:
you can check running the project `mobile-test-app` btw, it's only in React Native bindings that this would break ;)